.github/workflows/update_package_swift.yml

```diff
@@ -0,0 +1,148 @@
+name: Update Package.swift with checksum in S3 bucket
+
+on:
+  workflow_dispatch:
+    inputs:
+      force_build:
+        description: 'Force build even if sdk-version.txt has not changed'
+        required: false
+        default: 'false'
+  push:
+    branches:
+      - develop
+
+permissions:
+  contents: write
+
+jobs:
+  build:
+    runs-on: macos-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Check for sdk-version.txt Change
+        id: check_version
+        run: |
+          git fetch origin
+          if [ "${{ inputs.force_build }}" == 'true' ]; then
+            echo "🟠 Force build triggered. Continuing..."
+          elif git diff --quiet HEAD^ HEAD -- sdk-version.txt; then
+            echo "❌ No changes in sdk-version.txt. Exiting..."
+            exit 1
+          fi
+          echo "✅ sdk-version.txt has changed. Continuing..."
+
+      - name: Install dependencies
+        run: |
+          sudo xcode-select -s /Applications/Xcode_15.0.app
+          swift --version
+          echo "✅ Dependencies installed successfully"
+
+      - name: Build iOS Framework
+        run: |
+          xcodebuild archive \
+            -scheme CleverTapSDK \
+            -sdk iphoneos \
+            -archivePath ./build/CleverTapSDK-iOS \
```

Comment:
Consider adding the extension to the path, for example:
`  -archivePath ./build/CleverTapSDK-iphoneos.xcarchive \`

Reply:
done